wger/manager/tests/test_workout.py

```diff
@@ -123,6 +124,42 @@ def test_create_workout_logged_in(self):
         self.create_workout()
         self.user_logout()
 
+    def test_exporting_csv(self):
```

Comment:
Can you try `mocking` the exporting/saving of the csv file instead of creating the actual file. Check out the python mock library -> https://docs.python.org/3/library/unittest.mock.html